server/admin_management/api/app/schemas.py

```diff
@@ -44,6 +44,12 @@ class FamApplicationBase(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
 
+class FamApplicationGet(FamApplicationBase):
+    application_id: int
```

Comment:
I feel we need to return the application name as well, cause if we display in the dropdown, we have to display the name?

Reply:
Yes, indeed that's needed :) and it does have the application name because it is inherit from the `FamApplicationBase` which produces:
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/07d6ac50-58ae-4a05-80f9-21e7c1e43111)

Reply:
oh right, thanks Ian I missed that